docs/getting-started/brokers/sqs.rst

```diff
@@ -32,7 +32,9 @@ where the URL format is:
 
     sqs://aws_access_key_id:aws_secret_access_key@
 
-you must *remember to include the "@" at the end*.
+You must remember to include the ``@`` sign at the end and URL encode the
+password so it can be parsed correctly. This can be accomplished with
+builtin ``urllib.parse.quote`` function.
```

Comment:
would it be better to have an example of urllib.parse.quote?

Reply:
@pawelad @auvipy how about adding this, that will work both on Python 2 and 3:

```python
from kombu.utils.url import quote

quote('the/secret/key', safe='') # 'the%2Fsecret%2Fkey'
```

Reply:
One thing I was hesitant about was the use of f strings, but removing them would make that snippet twice as long - let me know what you think.